x-pack/plugin/enterprise-search/src/main/java/org/elasticsearch/xpack/enterprisesearch/index/ConnectorIndex.java

```diff
@@ -0,0 +1,101 @@
+
+
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.enterprisesearch.index;
+
+import org.elasticsearch.common.settings.Settings;
+
+public class ConnectorIndex {
+    public static final String MAPPING_JSON = "{\n" +
+        "  \"_meta\": {\n" +
```

Comment:
I _think_ the best practice is to use a builder here, see [this code for example](https://github.com/elastic/elasticsearch/blob/fc819609a16b366e17187e43c51287d77e082917/x-pack/plugin/watcher/src/main/java/org/elasticsearch/xpack/watcher/Watcher.java#L769). Although for practical purposes, it's fine!